packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -583,28 +583,6 @@ export class EndpointAccess {
  * Common configuration props for EKS clusters.
  */
 export interface ClusterProps extends ClusterOptions {
-  /**
-   * NOT SUPPORTED: We no longer allow disabling kubectl-support. Setting this
-   * option to `false` will throw an error.
-   *
-   * To temporary allow you to retain existing clusters created with
-   * `kubectlEnabled: false`, you can use `eks.LegacyCluster` class, which is a
-   * drop-in replacement for `eks.Cluster` with `kubectlEnabled: false`.
-   *
-   * Bear in mind that this is a temporary workaround. We have plans to remove
-   * `eks.LegacyCluster`. If you have a use case for using `eks.LegacyCluster`,
-   * please add a comment here https://github.com/aws/aws-cdk/issues/9332 and
-   * let us know so we can make sure to continue to support your use case with
-   * `eks.Cluster`. This issue also includes additional context into why this
-   * class is being removed.
-   *
-   * @deprecated `eks.LegacyCluster` is __temporarily__ provided as a drop-in
-   * replacement until you are able to migrate to `eks.Cluster`.
-   *
-   * @see https://github.com/aws/aws-cdk/issues/9332
-   * @default true
-   */
-  readonly kubectlEnabled?: boolean;
```

Comment:
I was having doubts about this since it might cause some minor breakage, but thought it's probably ok since it was deprecated...

Any objections?

Reply:
Sounds good